src/test/java/org/isf/patvac/test/Tests.java

```diff
@@ -215,6 +221,305 @@ public void testListenerShouldUpdatePatientToMergedWhenPatientMergedEventArrive(
 		assertThat(result.getPatient().getCode()).isEqualTo(mergedPatient.getCode());
 	}
 
+	@Test
+	public void testMgrGetPatientVaccineToday() throws Exception {
+		VaccineType vaccineType = testVaccineType.setup(false);
+		Vaccine vaccine = testVaccine.setup(vaccineType, false);
+		Patient patient = testPatient.setup(false);
+		PatientVaccine patientVaccine = testPatientVaccine.setup(patient, vaccine, true);
+
+		patientVaccine.setVaccineDate(new GregorianCalendar());
+
+		vaccineTypeIoOperationRepository.saveAndFlush(vaccineType);
+		vaccineIoOperationRepository.saveAndFlush(vaccine);
+		patientIoOperationRepository.saveAndFlush(patient);
+		patVacIoOperationRepository.saveAndFlush(patientVaccine);
+
+		ArrayList<PatientVaccine> patientVaccines = patVacManager.getPatientVaccine(false);
+		assertThat(patientVaccines.get(patientVaccines.size() - 1).getPatName()).isEqualTo(patientVaccine.getPatName());
+	}
```

Comment:
Could be bettere to enforce the "Today" word in the method's signature by persisting at least two objects in different dates?

Reply:
Good idea; added second object.